### PR TITLE
Snowfakery 2

### PR DESCRIPTION
**Pull request by prescod**
Release notes and dependency updates for Snowfakery 2.0

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ In the beginning, programmers created the databases. Now the databases were form
 
 And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.
 
+## Snowfakery 2.0
+
+Recipes can now merge in data from Salesforce Salesforce instances as
```
Comment (davidmreed) [inline HISTORY.md:9]:
```suggestion
Recipes can now merge in data from Salesforce orgs as
```

File: HISTORY.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ In the beginning, programmers created the databases. Now the databases were form
...
+records, IDs or datasets. (#395 and #401)
+
+Output streams can now be specified as 3rd party libraries. Any Python classname
```
Comment (davidmreed) [inline HISTORY.md:12]:
```suggestion
Output streams can now be specified as 3rd party libraries. Any Python class name
```

File: HISTORY.md [hunk 3]
```diff
@@ -4,6 +4,29 @@ In the beginning, programmers created the databases. Now the databases were form
...
+can be passed to --output-format as long as it adheres to the OutputStream
+protocol as shown in examples/YamlOutputStream.py (#351)
+
+Snowfakery is thread-safe/multi-processor-ready and available for parallel usage
+in CumulusCI. (CumulusCI Repo)
```
Comment (davidmreed) [inline HISTORY.md:17]: Did you intend this to be a link?

**Review by davidmreed — CHANGES_REQUESTED** [review]

File: HISTORY.md (hunk 3 above)
Comment (prescod) [reply HISTORY.md:17]: No, I think it would just clutter the doc for not enough benefit, IMO.

**Review by davidmreed — APPROVED** [review]

**Pull request merged**